cli.js

```diff
@@ -27,24 +27,55 @@ if (opts['strict-ssl'] === false) {
 }
 
 function flatten_configuration(cfg) {
+  const node_env = process.env.NODE_ENV || "development";
   let conf = extend(true, {}, cfg);
   let env = conf.ENV;
+
   if (env) {
+    Object.keys(env).forEach((envValue) => {
+      Object.keys(env[envValue]).forEach((fileName) => {
+        Object.keys(env[envValue][fileName]).forEach((prop) => {
+          if (prop === "override" && !env[envValue][fileName][prop]) {
```

Comment:
I think this is not handling the default value as `true`, do we actually want default to be `false`?

Reply:
I think I'm not quite understanding.
Using this [download_deps](https://github.com/hunterlester/safe_app_nodejs/blob/72d4e4938a9bbc3689503766e8ea1f27143ac691/package.json) JSON object, if I remove the `override` properties from lines 86 and 90, then those downloads override the default `safe_app` and `system_uri`, as expected.

Reply:
Ok, perfect, yes that's what I meant, so it's good.